src/rgw/store/sfs/sqlite/sqlite_users.cc

```diff
@@ -70,8 +83,8 @@ std::vector<DBOPUserInfo> SQLiteUsers::getUsersBy(Args... args) const {
 }
 
 std::string SQLiteUsers::getDBPath() const {
-  auto rgw_simplefile_path = ceph_context_->_conf.get_val<std::string>("rgw_simplefile_data_path");
-  auto db_path = std::filesystem::path(rgw_simplefile_path) / std::string(USERS_DB_NAME);
+  auto rgw_SFS_path = ceph_context_->_conf.get_val<std::string>("rgw_SFS_data_path");
```

Comment:
this one should be `rgw_sfs_data_path`, otherwise it crashes when accessing the DB path